### PR TITLE
chore!: updates to latest versions of packages

### PR DESCRIPTION
**Pull request by fyusuf-a**
- nestjs was given a minor update, but nothing that should impact the project (https://docs.nestjs.com/migration-guide)
- an `npm update` is performed before the start on the backend and the frontend, which updates the minor version of the packages
- `vuetify` is fixed to version `3.0.0-beta.10` to stay compatible with `vite-plugin-vuetify` which is fixed to version `1.0.0-alpha.17`

### CODE REVIEW

**Review by Mel-louie — COMMENTED** [review]
When I try to send messages in the Chat or come back to the Profile page after a game, I have this error:
```
Uncaught (in promise) TypeError: oldBindings[i] is undefined
    invokeDirectiveHook http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:3450
    patchElement http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:5427
    flushPostFlushCbs http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:1528
    flushJobs http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:1566
    promise callback*queueFlush http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:1476
(...)
```

**Comment by fyusuf-a** [comment]
> When I try to send messages in the Chat or come back to the Profile page after a game, I have this error:
> 
> ```
> Uncaught (in promise) TypeError: oldBindings[i] is undefined
>     invokeDirectiveHook http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:3450
>     patchElement http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:5427
>     flushPostFlushCbs http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:1528
>     flushJobs http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:1566
>     promise callback*queueFlush http://localhost:8000/node_modules/.vite/deps/chunk-B7LC2MEY.js?v=2268dbfd:1476
> (...)
> ```

I think my recent push-forced commit solves the issue.

**Comment by fyusuf-a** [comment]
Finally, `vuetify` is fixed to version `3.0.0-beta.12` and `vite-plugin-vuetify` to `1.0.0-alpha.17`.

**Review by Mel-louie — APPROVED** [review]
Previous bug is fixed now.
Everything seems to work fine.

Except for the warning which triggers when we go to a dynamic URL of a user that doesn't exist. Example: ```/profile/unexistingUser```
It's because child components, like MyStatistics, load even if the error is catch in OtherUsersProfile. I don't know how to solve it, but it's not a problem of your PR so I approve it and I will fix this bug

File: frontend/src/components/Profile/OtherUsersProfile.vue [hunk 1]
```diff
@@ -78,7 +78,6 @@
         } catch {
           this.$router.push({
             name: '404',
-            params: { path: '/404' },
           });
```
[the remark this answers is not in the document]
Comment (fyusuf-a) [reply frontend/src/components/Profile/OtherUsersProfile.vue:81]: You are right, I made the change and push-forced.

**Pull request merged**